homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -34,6 +40,35 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+SENSOR_STATUS_ON = [
+    "Panic",
+    "Motion",
+    "Light Detected",
+]
+
+SENSOR_STATUS_OFF = [
+    "End Panic",
+    "No Motion",
+    "Dark Detected",
+]
+
+SENSOR_STATUS_DEVICE_CLASS = {
+    "Panic": DEVICE_CLASS_SMOKE,
+    "End Panic": DEVICE_CLASS_SMOKE,
+    "Motion": DEVICE_CLASS_MOTION,
+    "No Motion": DEVICE_CLASS_MOTION,
+    "Light Detected": DEVICE_CLASS_LIGHT,
+    "Dark Detected": DEVICE_CLASS_LIGHT,
+}
```

Comment:
I think I'm going to make this be linked to to device type instead of status. Make more sense, and the status may not be available on a restore.